packages/salesforcedx-vscode-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -354,26 +354,20 @@ export class ApexReplayDebug extends LoggingDebugSession {
     }
   }
 
-  public getHandlerForDebugConsole(): (message: string) => void {
-    if (this.traceAll || this.trace.indexOf(TRACE_CATEGORY_LOGFILE) !== -1) {
-      return (message: string) => {
-        this.printToDebugConsole(message);
-      };
-    } else {
-      // tslint:disable-next-line:no-empty
-      return (message: string) => {};
-    }
+  public shouldTraceLogFile(): boolean {
+    return this.traceAll || this.trace.indexOf(TRACE_CATEGORY_LOGFILE) !== -1;
   }
 
   public printToDebugConsole(
-    msg?: string,
+    msg: string,
+    category = 'stdout',
```

Comment:
Usually default arguments go last. I can't find anything about it in JavaScript but I think the same wisdom from C++ apply on why having it last is better. See https://stackoverflow.com/questions/1123725/why-should-default-parameters-be-added-last-in-c-functions